plugins/modules/win_package.py

```diff
@@ -256,6 +256,34 @@
     state: present
     log_path: D:\logs\vcredist_x64-exe-{{lookup('pipe', 'date +%Y%m%dT%H%M%S')}}.log
 
+- name: Install Application from msi with multiple properties for installer
+  ansible.windows.win_package:
+    path: C:\temp\Application.msi
+    state: present
+    arguments: >-
+      /quiet
+      /norestart
+      /log
+      C:\application_server.log
+      SERVICE=1
+      DBNAME=ApplicationDB
+      DBSERVER=.\SQLEXPRESS
+      INSTALLDIR="C:\Program Files (x86)\App lication\App Server"
+
+- name: Install Microsoft® SQL Server® 2019 Express (DPAPI example)
+  ansible.windows.win_package:
+    path: C:\temp\SQLEXPR_x64_ENU\SETUP.EXE
+    product_id: Microsoft SQL Server SQL2019
+    arguments:
+      - SAPWD=VeryHardPassword
+      - /ConfigurationFile=C:\temp\configuration.ini
+  become: yes
+  become_flags: logon_type=batch
```

Comment:
Any reason for this particular logon flag?

Reply:
Took some time to verify. It did work actually without the flags as well.